content/articles/an-overview-of-web-3-0/index.md

```diff
@@ -0,0 +1,105 @@
+The World wide web is a crucial element of our modern life. It has become part of us, and it would not be easy to outgrow our lives without it. This is because technology has greatly advanced as people can efficiently work remotely and easily communicate with the help of the internet. The reliance on the world wide web has grown in the last three decades and has gone through stages of evolution from [Web 1.O](https://websitebuilders.com/how-to/glossary/web1/), [Web 2.O](https://en.wikipedia.org/wiki/Web_2.0), and [Web 3.O](https://web3.foundation/).
```

Comment:
World Wide Web and not World wide web, also consider adding (WWW) which much popular.